src/test/resources/features/refapp-2.x/03-trackPatient/EditPatientRelationship.feature

```diff
@@ -0,0 +1,13 @@
+ Feature: EditRelationshipType
+ 
+ Given System loads clinicianFacingPatientDashboardPage
+    When  User clicks on editRegistrationInformation
```

Comment:
there's a formatting issue here, check existing features to know how to format background and user scenarios

Reply:
Yeah formatting issues , my ide is now becoming a serious problem  thanks